transport.go

```diff
@@ -93,7 +94,7 @@ type transport struct {
 var _ tpt.Transport = &transport{}
 
 // NewTransport creates a new QUIC transport
-func NewTransport(key ic.PrivKey, psk pnet.PSK, filters *filter.Filters) (tpt.Transport, error) {
+func NewTransport(key ic.PrivKey, psk pnet.PSK, connGater connmgr.ConnectionGater) (tpt.Transport, error) {
```

Comment:
Let's call this `gater` everywhere; there's no other gater in the system, and we don't expect there to be, so the `Connection` qualifier is redundant. (And it's idiomatic Go to shorten variable names the nearer they're used).

Reply:
This is done.